auth/app/Http/Controllers/Auth/RegisteredUserController.php

```diff
@@ -49,6 +49,13 @@ public function store(Request $request)
 
         event(new Registered($user));
 
+        // Create user in API Database.
+        (new User([
+            'first_name' => $request->first_name,
+            'last_name' => $request->last_name,
+            'email' => $request->email,
+        ]))->setConnection('pgsql_api')->save();
+
```

Comment:
I see what you're trying to do, but I'm not sure this is where we want to make this connection. Definitely doesn't need to be part of this PR.